docs/php/symfony/task-manager/index.mdx

```diff
@@ -7,6 +7,12 @@ import {LinkList} from "../../../../src/components/Link/LinkList";
 	packagist="https://packagist.org/packages/21torr/task-manager"
 />
 
+The task manager is a bundle that builds in top of [Symfony Messenger] to add additional convenience functionality for running, logging and queuing tasks.
+
+:::note
+As this bundle adds quite a lot of functionality on top of Symfonys messenger system, we use the names "task" instead of "message" and "task handler" instead of "message handler" to make clear, that these are the *extended versions*.
```

Comment:
```suggestion
As this bundle adds quite a lot of functionality on top of Symfonys messenger system, we use the names "task" instead of "message", and "task handler" instead of "message handler" to make clear, that these are the *extended versions*.
```